badge.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: none
+permalink: /badge/
+---
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons">
+<link rel="stylesheet" href="https://unpkg.com/bootstrap-material-design@4.1.1/dist/css/bootstrap-material-design.min.css" integrity="sha384-wXznGJNEXNG1NFsbm0ugrLFMQPWswR3lds2VeinahP8N0zJw9VWSopbjv2x7WCvX" crossorigin="anonymous">
```

Comment:
Do we really need to include bootstrap material? Isn't it already in our static folder?

Reply:
Yeah because this is a page of `layout: none`. So no CSS at all will be loaded.